packages/flutter_tools/lib/src/bundle_builder.dart

```diff
@@ -137,16 +139,18 @@ Future<AssetBundle?> buildAssets({
 Future<void> writeBundle(
   Directory bundleDir,
   Map<String, AssetBundleEntry> assetEntries, {
-  Logger? loggerOverride,
```

Comment:
Interestingly, this was only partially used. I wonder if there was a bug here. I deemed it not worthy of further investigation.